### PR TITLE
Fix downgrade prio of job actually increasing it.

### PR DESCRIPTION
**Pull request by bitope**

### CODE REVIEW

File: Assets/Scripts/Models/JobQueue.cs [hunk 1]
```diff
@@ -66,4 +66,12 @@ public void Remove(Job j)
         jobQueue.RemoveAt(jobQueue.IndexOfValue(j));
     }
 
+    public IEnumerable<Job> PeekJobs()
+    {
+        // For debugging only. For the real thing we want to return something safer (like preformatted strings.).
+        foreach (var job in jobQueue.Values)
```
Comment (MANDAL0R3) [inline Assets/Scripts/Models/JobQueue.cs]: According to CONTRIBUTING.md, we are not using 'var' unless it can't be helped.

**Comment by RodRitter** [comment]
Please sort out the `var` issue, and then I can merge this.

**Comment by bitope** [comment]
done

**Comment by RodRitter** [comment]
Nice. Merged.

**Pull request merged**